web/client/utils/mapinfo/wfs.js

```diff
@@ -27,6 +27,21 @@ import assign from 'object-assign';
  * @return {object} an object with `request`, containing request params, `metadata` with some info about the layer and the request, and `url` to send the request to.
  */
 const buildRequest = (layer, { map = {}, point, currentLocale, params, maxItems = 10 } = {}, infoFormat, viewer, featureInfo) => {
+    const { features = [] } = point?.intersectedFeatures?.find(({ id }) => id === layer.id) || {};
+    if (features.length) {
+        return {
+            request: {
+                features: [...features],
+                outputFormat: 'application/json'
+            },
+            metadata: {
+                title: isObject(layer.title)
+                    ? layer.title[currentLocale] || layer.title.default
+                    : layer.title
+            },
```

Comment:
here are missing the metadata available in the other request: regex, viewer and featureInfo; so please add them, thanks